docs/UserGuide.md

```diff
@@ -300,35 +355,34 @@ Examples:
 
 ### Adding a lesson: `add -l`
 
-Adds a new lesson to TutorAid.
+Adds a new lesson to TutorAid. The lesson's capacity, price and timing are optional details for tutors to include.
```

Comment:
Just a minor nitpick here! Can considering changing from `tutors` to `you` to make it more personalised